### ISSUE TITLE
Make a State-ful LSTM with data input from TF Dataset

### ISSUE THREAD

**Issue by Gumilton** [tensorflow/tensorflow#15088]
#14906 
This is to reference a closed issue.
Thanks for @mrry suggestion on using Dataset.flat_map to slice the input signal sequence.
However, how could we know the beginning and ending of the original sequence after sliced?
And how could we reset the LSTM state so that we can make a state-ful LSTM?
Thanks!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by mrry** [comment]
You should be able to use the existing [`tf.nn.static_state_saving_rnn()`](https://www.tensorflow.org/api_docs/python/tf/nn/static_state_saving_rnn) function with a `tf.data` input pipeline and a [`tf.contrib.training.SequenceQueuingStateSaver`](https://www.tensorflow.org/api_docs/python/tf/contrib/training/SequenceQueueingStateSaver). Note that the latter relies on queues and queue-runners, so you'll still need to call `tf.train.start_queue_runners()`.

**Comment by Gumilton** [comment]
@mrry Thank you very much! I am looking into it and will come back and update once succeed.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by mrry** [comment]
Hopefully you succeeded! I'm closing this issue now, but feel free to follow up on Stack Overflow if you have other usage questions.
